src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php

```diff
@@ -165,6 +165,32 @@ public function tolerates(array $deprecationGroups)
         return true;
     }
 
+    /**
+     * @param array<string,DeprecationGroup> $deprecationGroups
+     *
+     * @return bool true if the thresholds is not reached for the deprecation type
+     */
+    public function toleratesGroup(string $groupName, array $deprecationGroups): bool
```

Comment:
I'd name this "toleratesForGroup" since it's not the group that is tolerated, but some amount of deprecation for that group.